tests/unit/test_util/test_security.py

```diff
@@ -0,0 +1,304 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2022 NV Access Limited.
+
+"""Unit tests for the blockUntilConditionMet submodule.
+"""
+
+from dataclasses import dataclass
+from typing import (
+	List,
+	Optional,
+)
+import unittest
+from unittest.mock import patch
+
+from utils.security import (
+	_WindowNotFoundError,
+	_isWindowAboveWindowMatchesCond,
+)
+import winUser
+
+
+@dataclass
+class _MoveWindow:
+	"""Used to move a window from one index to another when a specific index is reached."""
+	startIndex: int  # A window at this index
+	endIndex: int  # is moved to this index

```

Comment:
If using indexes for this, it would clearer if this was 'moveFromIndex` / `moveToIndex`

Reply:
I'm moving using HWNDs instead now